src/components/apply/formNormalizers.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+export const stripSocialSec = input => _.replace(input, new RegExp('-','g'), '');
+const middleInitialRegEx = new RegExp('[a-zA-Z]?');
+
+export const maxLengthNormalizer = len => (input, prevInput) => {
```

Comment:
There seems to be a pattern to this code, see if you can abstract something to get rid of the repetition.